app/main.py

```diff
@@ -32,6 +32,7 @@
 app.config['SQLALCHEMY_DATABASE_URI'] = db_connenction_url
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['PROPAGATE_EXCEPTIONS'] = True
+app.config['JSON_SORT_KEYS'] = False
```

Comment:
Also, do you think this is needed? If so, it's not being used in the current logic. Please see.

Reply:
Yes, it is needed. Even though you make it pdl.py using the json with correctly order, flask sorts again if this is not there.

Reply:
You can test it in your local, removing this line. Then you will see the returned json starts from Alabama instead of texas

Reply:
Oh! Okay. Thanks. I thought this was regular config. Now I realized that it's a Flask configuration.